diff -up apsw-3.46.0.1/src/apsw.c.1~ apsw-3.46.0.1/src/apsw.c
--- apsw-3.46.0.1/src/apsw.c.1~	2024-06-16 21:44:24.000000000 +0200
+++ apsw-3.46.0.1/src/apsw.c	2024-07-07 18:47:30.031848579 +0200
@@ -279,6 +279,7 @@ get_apsw_version(void)
 static PyObject *
 enable_shared_cache(PyObject *Py_UNUSED(self), PyObject *const *fast_args, Py_ssize_t fast_nargs, PyObject *fast_kwnames)
 {
+#ifdef INCLUDE_DEPRECATED
   int enable = 0, res;
   {
     Apsw_enable_shared_cache_CHECK;
@@ -291,6 +292,7 @@ enable_shared_cache(PyObject *Py_UNUSED(
 
   if (res != SQLITE_OK)
     return NULL;
+#endif
 
   Py_RETURN_NONE;
 }
